sdcm/nemesis.py

```diff
@@ -3915,6 +3947,13 @@ def disrupt(self):
         self.disrupt_toggle_table_ics()
 
 
+class ToggleGcModeMonkey(Nemesis):
+    kubernetes = True
+
+    def disrupt(self):
```

Comment:
what happens if this nemesis is executed against Scylla that does not support this feature?

Reply:
I think it should be addressed by correct and limited backports. isn't it?

Reply:
is this feature in 2022 too (i see you set back port to 5.0)?
if yes, then you are right, the branches will dictate in which versions it may run, but a very simple check inside the nemesis itself can just skip it, if it is older than the supported versions, no?

Reply:
We already have a way to skip nemesis in case of unsupported version (see decorator for `disrupt_show_toppartitions`).
Consider the case when we want to run some test on 4.6 but still using master branch (e.g. someone forgotten about new features nemesis, or just needs to run with master sct).

Reply:
Well this is a bit funny - if we control everything with supported-version decorators then we don't need version branches do we?

i don't mind adding it, just need some confirmation from maintainers - what's the updated convention about it.